From 2851d00861391c1f8c39002e56e5adce8ca37a61 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rainer=20M=C3=BCller?= <raimue@macports.org>
Date: Tue, 8 Nov 2016 18:18:12 +0100
Subject: [PATCH 1/3] Makefile: support DESTDIR variable for install

The DESTDIR variable is commonly used to install to a separate root
directory. This is often leveraged to create packages for package
managers.
---
 Makefile | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/Makefile b/Makefile
index 220a7e5..2564c07 100644
--- a/Makefile
+++ b/Makefile
@@ -47,25 +47,25 @@ endif
 
 install: install-lib install-doc
 
-install-lib: $(INSTALL_EXT)
-	install -C -m 0755 $(LIBS) $(INSTALL_LIB)/
-	install -d -m 0755 $(INSTALL_EXT)/
-	install -C -m 0755 $(EXTS) $(INSTALL_EXT)/
+install-lib: $(DESTDIR)$(INSTALL_EXT)
+	install -C -m 0755 $(LIBS) $(DESTDIR)$(INSTALL_LIB)/
+	install -d -m 0755 $(DESTDIR)$(INSTALL_EXT)/
+	install -C -m 0755 $(EXTS) $(DESTDIR)$(INSTALL_EXT)/
 
 install-doc:
-	install -d -m 0755 $(INSTALL_MAN1)
-	install -C -m 0644 $(MAN) $(INSTALL_MAN1)
+	install -d -m 0755 $(DESTDIR)$(INSTALL_MAN1)
+	install -C -m 0644 $(MAN) $(DESTDIR)$(INSTALL_MAN1)
 
 uninstall: uninstall-lib uninstall-doc
 
 uninstall-lib:
-	rm -f $(INSTALL_CMD)
-	rm -fr $(INSTALL_EXT)
+	rm -f $(DESTDIR)$(INSTALL_CMD)
+	rm -fr $(DESTDIR)$(INSTALL_EXT)
 
 uninstall-doc:
-	rm -f $(INSTALL_MAN1)/$(NAME).1
+	rm -f $(DESTDIR)$(INSTALL_MAN1)/$(NAME).1
 
-$(INSTALL_EXT):
+$(DESTDIR)$(INSTALL_EXT):
 	mkdir -p $@
 
 clean purge:
-- 
2.10.1

